src/composables/use-fetch-state.ts

```diff
@@ -24,7 +24,6 @@ const statuses = Object.freeze({
   FETCHING: 'fetching',
   SUCCESS: 'success',
   ERROR: 'error',
-  FINISHED: 'finished',
```

Comment:
Can we remove the comment above describing the `finished` state, if it no longer exists?

```
 * - `finished`: for multi-page requests, this is true when no more pages are left.
```